res/datasets/schools/format.py

```diff
@@ -0,0 +1,12 @@
+import csv
+import re
+# uses csv and regex, maybe converting to a dict is better or something? not sure
+
+with open("TDCSB_schools.csv","r") as f:
```

Comment:
With a dataset, you don't need to do this. Since the TCDSB schools are imported as a panda data frame in the preprocessing jupyter notebook, the `geometry` column is an object. You would be able to access the co-ordinates by doing something along the lines of `df['geometry'].coordinates[0] for longitude in this case, and [1] for latitude.

You can then export it to CSV: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html

Reply:
got it. will do after we have the meeting.